packages/flutter_tools/lib/src/resident_devtools_handler.dart

```diff
@@ -144,13 +144,22 @@ class FlutterResidentDevtoolsHandler implements ResidentDevtoolsHandler {
       return;
     }
     try {
-      await _invokeRpcOnFirstView(
-        'ext.flutter.connectedVmServiceUri',
-        device: device,
-        params: <String, dynamic>{
-          'value': uri.toString(),
-        },
-      );
+      if (device.targetPlatform == TargetPlatform.web_javascript) {
```

Comment:
since _invokeRpcOnFirstView is handling the web_javascript case directly, can we remove this?

Reply:
Ahh right, good catch